test/widget_test.dart

```diff
@@ -20,4 +16,24 @@ void main() {
   {
     expect(songs[0].lyrics, "test lyrics");
   });
+
+  test('Get author name', ()
+  {
+    expect(authors[0].name, 'Foo Author');
+  });
+
+  test('Get song count by author', ()
+  {
+    expect(authors[1].songs, 58);
+  });
+
```

Comment:
Create a separate class AuthorService_test in test/service folder  and move this test to there.